modules/social_features/social_swiftmail/social_swiftmail.module

```diff
@@ -189,3 +181,28 @@ function social_swiftmail_element_info_alter(array &$info) {
     ];
   }
 }
+
+/**
+ * Implements hook_mailer_builder_info_alter().
+ */
+function social_swiftmail_mailer_builder_info_alter(array &$email_builders): void {
+  if (isset($email_builders['user'])) {
+    $email_builders['user']['class'] = 'Drupal\social_swiftmail\Plugin\EmailBuilder\UserEmailBuilder';
+  }
+}
+
+/**
+ * Implements hook_mailer_post_render().
+ */
+function social_swiftmail_mailer_post_render(EmailInterface $email): void {
+  $css_inliner = new CssToInlineStyles();
+  $body = (string) $email->getHtmlBody();
+  $email->setHtmlBody($css_inliner->convert(check_markup($body, 'full_html')));
+
+  $dsn = $email->getTransportDSN();
+
+  if (strpos($dsn, 'null://file-spool:/') === 0) {
+    $email->addTextHeader('X-Transport', 'FileSpool');
+    $email->addTextHeader('X-Spool-Directory', substr($dsn, 19));
+  }
+}
```

Comment:
What is happening here? :D

Reply:
Good question! This was a piece of code that was already in there. Double checked it
- Css inline is a way to get optimisation for css includes into the email that is being sent. For the old swiftmailer this was part of their messageBody where they did this action. In Symfony mailer they also use this but it's already part of the mailer policy that we have set (see attachment)
![Screenshot 2023-10-31 at 13 15 03](https://github.com/goalgorilla/open_social/assets/19951171/025e597f-85f1-49c9-bd6f-8a67fbaed6ad)
- When have set the mailer transport to null it will add these header. We could move this to the log_mail plugin, but I want to remove the code as I don't see why it would be needed.

Therefore conclusion. I will remove this whole function as I believe it's not needed.